Examples/apps/ObjCExample/ObjCExample/AppDelegate.m

```diff
@@ -18,13 +18,15 @@ @implementation AppDelegate
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
     // Override point for customization after application launch.
-    SEGConfiguration *config = [[SEGConfiguration alloc] initWithWriteKey:@"WRITE_KEY"];
+    SEGConfiguration *config = [[SEGConfiguration alloc] initWithWriteKey:@"<writekey>"];
     config.trackApplicationLifecycleEvents = YES;
     
-    SEGAnalytics *analytics = [[SEGAnalytics alloc] initWithConfiguration: config];
+    _analytics = [[SEGAnalytics alloc] initWithConfiguration: config];
```

Comment:
Does this get assigned to self.analytics?

Reply:
yup.